### PR TITLE
Adding CSEs to CE team list!

### PR DESCRIPTION
**Pull request by virginiaulrich**
@DaedalusG @northyg @adeola-ak I have added you all -- Adeola, we can add a link to your team directory listing once you've done that update :)

### CODE REVIEW

**Review by DaedalusG — APPROVED** [review]
Its official!

File: handbook/ce/index.md [hunk 1]
```diff
@@ -60,9 +60,13 @@ CSEs are the go-to technical team for our CEs, helping customers both pre- and p
   - [Mike McLaughlin](../../company/team/index.md#mike-mclaughlin-he-him)
   - [Alex Fogg](../../company/team/index.md#alex-fogg-he-him)
   - T. (Director of Customer Engineering, US East; starting 2021-03-01)
-- Customer Support
+- Customer Support Engineers
+  - Adeola Akinsiku
+  - [Warren Gifford](../../company/team/index.md#giselle-northy-she-her)
```
Comment (northyg) [inline handbook/ce/index.md:65]: Looks like the markdown links for Warren and Giselle are not pointing to the correct profiles.

**Review by northyg — CHANGES_REQUESTED** [review]
Could you please make sure Giselle and Warrens links point to the correct profile?

File: handbook/ce/index.md (hunk 1 above)
Comment (DaedalusG) [reply handbook/ce/index.md:65]: Oh shoot good catch!

Comment (virginiaulrich) [reply handbook/ce/index.md]: Thanks for catching that, Giselle! It's like I was trying to make a point about why PR and I wasn't 🤣 .... but also totally why a second set of 👀 is gold ✨

**Review by DaedalusG — APPROVED** [review]

**Review by DaedalusG — APPROVED** [review]

**Review by northyg — APPROVED** [review]

**Pull request merged**